sdk/test-utils/recorder-new/MIGRATION.md

```diff
@@ -0,0 +1,160 @@
+# Migration Guide
+
```

Comment:
Is there an explanation somewhere on why the new recorder is called "Unified"?

Reply:
I believe it’s called unified because there’s a single provider for all languages. There could be a better name though.

Reply:
I am not questioning the name, just wonder whether we should have some more info here or linking to other docs.

Reply:
I can definitely add a short thing here about it, but this shouldn't be the only place I think.